test/Bootstrapper.FunctionalTests/BootstrapperTestUtils.cs

```diff
@@ -31,43 +31,5 @@ public static class BootstrapperTestUtils
             stdErr = stdErrStr;
             return exitCode;
         }
-
-        public static DisposableDir PrepareTemporarySamplesFolder(string runtimeHomeDir)
```

Comment:
I am moving this method to a more common util class, which is accessible in a bigger scope.